config/checkstyle/checkstyle.xml

```diff
@@ -201,11 +201,6 @@
     </module>
     <module name="OverloadMethodsDeclarationOrder"/>
     <module name="VariableDeclarationUsageDistance"/>
-    <module name="CustomImportOrder">
-      <property name="sortImportsInGroupAlphabetically" value="true"/>
-      <property name="separateLineBetweenGroups" value="true"/>
-      <property name="customImportOrderRules" value="THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE###STATIC"/>
-    </module>
```

Comment:
This commit is my doing.

This check is EXCESSIVELY pedantic and doesn't even follow IntelliJ's own ordering, so it's got to go.

No matter the objections, if this is the only point I'll approve.